src/stores/widgets/SpectralProfileWidgetStore.ts

```diff
@@ -26,6 +27,7 @@ export class SpectralProfileWidgetStore extends RegionWidgetStore {
     @observable lineWidth: number;
     @observable linePlotPointSize: number;
     @observable linePlotInitXYBoundaries: { minXVal: number, maxXVal: number, minYVal: number, maxYVal: number };
+    @observable smoothingStore: ProfileSmoothingStore;
```

Comment:
this can be a `readonly` variable without an observable decorator, can't it? since it's only assigned once